src/Lykke.Job.TradeDataAggregator/Modules/JobModule.cs

```diff
@@ -11,19 +11,20 @@
 using Lykke.Job.TradeDataAggregator.Core.Domain.Exchange;
 using Lykke.Job.TradeDataAggregator.Core.Domain.Feed;
 using Lykke.Job.TradeDataAggregator.Core.Services;
-using Lykke.Service.Assets.Client.Custom;
 using Microsoft.Extensions.DependencyInjection;
 using Lykke.Job.TradeDataAggregator.Services;
+using Lykke.Service.Assets.Client;
+using Lykke.SettingsReader;
 
 namespace Lykke.Job.TradeDataAggregator.Modules
 {
     public class JobModule : Module
     {
-        private readonly AppSettings _settings;
+        private readonly IReloadingManager<AppSettings> _settings;
         private readonly ILog _log;
         private readonly ServiceCollection _services;
 
-        public JobModule(AppSettings settings, ILog log)
+        public JobModule(IReloadingManager<AppSettings> settings, ILog log)
```

Comment:
Will be better to pass `DbSettings` wrapped into the `IReloadingManager`, but `AppSettings` as raw, unwrapped object